### PR TITLE
Create graph from files beta

### PR DESCRIPTION
**Pull request by muhammadshoaib**
Two functions have been created that can be used as following to load vertex and edges from csv files. 

```
CREATE FUNCTION ag_catalog.load_labels_from_file(graph_name name, label_name name, file_path text)

CREATE FUNCTION ag_catalog.load_edges_from_file(graph_name name, label_name name, file_path text)
```

### CODE REVIEW

File: src/backend/utils/load/ag_load_edges.c [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+    cr->curr_row_length = 0;
+    cr->row += 1;
+}
+
+static int is_space(unsigned char c) {
+    if(c == CSV_SPACE || c == CSV_TAB) return 1;
+    return 0;
+}
+
+/*
```
Comment (JoshInnis) [inline src/backend/utils/load/ag_load_edges.c:122]: This should be removed

**Review by jrgemignani — COMMENTED** [review]
Should we be modifying .gitignore ?

File: src/include/utils/load/csv.h [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+*/
+
+#ifndef LIBCSV_H__
+#define LIBCSV_H__
+#include <stdlib.h>
+#include <stdio.h>
+
+#ifdef __cplusplus
+extern "C" {
```
Comment (emotionbug) [inline src/include/utils/load/csv.h:27]: i think it need be indent.

File: src/include/utils/load/csv.h [hunk 3]
```diff
@@ -0,0 +1,108 @@
+//
+// Created by Shoaib on 12/5/2021.
```
Comment (emotionbug) [inline src/include/utils/load/csv.h:2]: remove this line

File: regress/sql/age_load.sql [hunk 4]
```diff
@@ -0,0 +1,57 @@
+\! cp -r regress/age_load/data regress/instance/data/age_load
```
Comment (emotionbug) [inline regress/sql/age_load.sql:1]: is it need?

File: src/include/utils/load/csv.h (hunk 2 above)
Comment (muhammadshoaib) [reply src/include/utils/load/csv.h:27]: This is original library so I didn't changed anything in this one.

File: regress/age_load/age_load_test.sql [hunk 5]
```diff
@@ -0,0 +1,34 @@
+DROP SCHEMA IF EXISTS agload_test_graph CASCADE ;
```
Comment (emotionbug) [inline regress/age_load/age_load_test.sql:1]: It's not even a test case. Do we need this?

File: src/backend/utils/load/ag_load_edges.c [hunk 6]
```diff
@@ -0,0 +1,190 @@
...
+#include <errno.h>
+#include <stdlib.h>
+#include <unistd.h>
+
+#include "utils/load/csv.h"
+#include "utils/load/ag_load_edges.h"
+#include "utils/load/age_load.h"
+
+
+void edge_field_cb(void *field, size_t field_len, void *data) {
```
Comment (emotionbug) [inline src/backend/utils/load/ag_load_edges.c:31]: I don't know exact bracket rules in this project. but, never used like this.

File: src/include/utils/load/ag_load_labels.h [hunk 7]
```diff
@@ -0,0 +1,105 @@
...
+#include "catalog/ag_graph.h"
+#include "catalog/ag_label.h"
+#include "commands/label_commands.h"
+#include "utils/ag_cache.h"
+#include "utils/agtype.h"
+#include "utils/graphid.h"
+
+
+#define AGE_VERTIX 1
+#define AGE_EDGE 2
```
[the remark this answers is not in the document]
Comment (emotionbug) [reply src/include/utils/load/ag_load_labels.h:70]: + with counts struct.

**Pull request merged**